tests/integration_test/services_test.py

```
@@ -304,6 +321,661 @@
     assert not response["success"]
 
 
+
+
+import calendar
```

Ruff (0.12.2)

Redefinition of unused `calendar` from line 72

Remove definition: `calendar`

(F811)